kano_updater/scenarios.py

```diff
@@ -361,6 +361,7 @@ def beta_210_to_beta_220(self):
 
     def beta_220_to_beta_230(self):
         install('rsync')
+        run_cmd_log('kano-apps install --no-gui powerup')
```

Comment:
I think you need sudo here since the install is non-interactive. It won't be able to prompt you for a password.